sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -83,12 +80,6 @@
   // List of recorded events.
   @GuardedBy("lock")
   private final EvictingQueue<TimedEvent> events;
-  // Number of events recorded.
-  @GuardedBy("lock")
-  private int totalRecordedEvents = 0;
```

Comment:
This should be used in the SpanData see https://github.com/open-telemetry/opentelemetry-proto/blob/master/opentelemetry/proto/trace/v1/trace.proto#L35 for where in the SpanData is used.